scripts/configure.py

```diff
@@ -99,6 +99,7 @@
             'sudo systemctl restart rsyslog.service',
 
             'echo "reconfiguring journald to only log to RAM"',
+            'sudo chmod 666 /etc/systemd/journald.conf',
```

Comment:
```suggestion
```

I feel pretty strongly we shouldn't modify permissions on this file, and instead use privilege escalation if we want to poke it (ie, "go through the front door" - use `sudo` or similar in a `subprocess`.)